lib/events.js

```diff
@@ -351,7 +351,7 @@ EventEmitter.prototype.prependOnceListener =
       return this;
     };
 
-// emits a 'removeListener' event if and only if the listener was removed
+// emits -> Emits a 'removeListener' event if and only if the listener was removed.
```

Comment:
Almost 😄 
I believe what @Trott ment with the arrow is `emits` needs to be replaced with `Emits`.
I'll fix this.